### DISCUSSION TITLE
Platformer Test

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#7] [Games]
# Platformer Test

https://wasm4.org/play/platformer-test/?giscus=70ffae2203e2cac8354fbde49i%2Ffcc5ql2VeddpUpREid1Iockq%2FBtUFpl0l1VZ24b3h0B5CWr%2FAlqK7MOIziFCyTYIrqoZo9ox1frcJUwywTFC6Py%2BqgNAvWuamloszHz%2B11l23lXeVhq2%2BDcs%3D

**Comment by aduros** [comment]
A simple platformer experiment, might turn it into a real game later.

[Source code](https://github.com/aduros/wasm4/tree/main/examples/platformer-test)

**Reply by triptych** [reply]
It's cool!
